skbase/lookup/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+"""Tests of skbase lookup functionality."""
```

Comment:
do we need `sktime` BSD license blurb in some places here?

Reply:
Added it to test_lookup.py, which is where the tests that are affected live.